.clang-format

```diff
@@ -2,9 +2,8 @@ IndentWidth: 4
 ContinuationIndentWidth: 4
 Language: Cpp
 UseTab: Never
-ColumnLimit: 120
+ColumnLimit: 0
```

Comment:
Can't we get away with a longer limit like 160 or something? I do like having not infinite line length.

Reply:
After the discussion with @pixel-stuck in discord, I think we should try something like 160 and see if we can get away with not having to put struct comments on their own line. Also we're both in agreement about not spacing struct fields apart and having one line per field with no gaps.